### PR TITLE
OPSEXP-2365 Update docs for v7.0.0

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2365

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,16 +100,11 @@ The table below shows the exact version of ACS deployed with each chart version/
...
-version to a new namespace rather than attempting upgrades.
-
-> Deploying to new namespace is always the preferred way of upgrading ACS as we
-> do not test charts for upgrade scenarios (even with previous versions)
-> neither do we provide roll-back facilities.
+During the development of 5.4.0 we've started moving individual components
+templates (search, sync, activemq, ...) into individual charts on
+[alfresco-helm-charts](https://github.com/Alfresco/alfresco-helm-charts). That
+decision introduced some breaking changes like resource renaming and values
+structure modifications that has been shipped since v6.0.0.
```
Comment (gionn) [inline README.md:107]: I make it more compact but after the new major may not even worth mentioning

File: README.md [hunk 2]
```diff
@@ -6,50 +6,55 @@
...
+leverage the new individual subcharts we started releasing since March 2023 in a
+dedicated repo:
+[alfresco-helm-charts](https://github.com/Alfresco/alfresco-helm-charts).
+
+This change will make life easier to whoever want to customize or extend the
+helm deployments and simplify the future maintenance, at the cost of breaking
+the compatibility with the values structure which remained almost stable since
+the v6 release.
+
+Please review the new [values](helm/alfresco-content-services/values.yaml)
```
Comment (alxgomz) [inline README.md:28]: Also link the upgrade guide here?

File: README.md [hunk 3]
```diff
@@ -120,7 +120,7 @@ file for a version with the latest charts or using the old charts.
 | 6.2                     | 5.3.0                           |
 | 6.1                     | 5.1.1                           |
 
-> These charts are mentioned for reference but are not supported.
+> These charts should not be used for any new deployment but just for reference.
```
Comment (alxgomz) [inline README.md:123]: 👍

**Review by alxgomz — APPROVED** [review]
That was very much needed! 👍

**Pull request merged**